Manifest.toml

```diff
@@ -1,8 +1,12 @@
+<<<<<<< HEAD
```

Comment:
It should work if you remove
``<<<<<<< HEAD``
``=======``
``>>>>>>> 8113743ed648854aa13876e9cf2469d1ce71c4a6``